Common/src/Pollster/Pollster.cs

```diff
@@ -151,6 +151,11 @@ await foreach (var message in messages.WithCancellation(cancellationToken)
 
           var combinedCts = CancellationTokenSource.CreateLinkedTokenSource(message.CancellationToken,
                                                                             cancellationToken);
+          if (combinedCts.IsCancellationRequested)
+          {
+            return;
```

Comment:
Do we really want a return? or throw a `OperationCanceledException`?

Reply:
Yes, at this point we do not want to acquire the task. So we quit. Maybe break would have been more appropriate. and put the message at postponed

Reply:
well, break break because there is a foreach and a while

Reply:
I really don't like the break; break;

Reply:
return est pas si mal du coup

(ps: rigolo la meme discussion a eu lieu sur un autre langage que wil "adore" :)

PS:  on a ete bien conditionné à éviter l'utilisation des goto...